File: 08 - Pizza Party/Haskell/main.hs
main :: IO()

main = do
    print("How many people? ")
    noPeople <- readLn :: IO Int
    print("How many pizzas? ")
    noPizza <- readLn :: IO Int
    print("How many slices per pizza? ")
    noSlices <- readLn :: IO Int

    let (slicesPer, slicesLeft) = divMod (noPizza * noSlices) noPeople

    print(show noPeople ++ " people with " ++ show noPizza ++ " pizzas, each of " ++ show noSlices)
    print("Each person gets " ++ show slicesPer ++ " slices of pizza.")
    print("There are " ++ show slicesLeft ++ " leftover pieces.")

File: 08 - Pizza Party/Golang/main.go
package main

import "fmt"

func main() {
	var noPeople int
	fmt.Printf("How many people? ")
	fmt.Scanf("%d", &noPeople)

	var noPizza int
	fmt.Printf("How many pizzas? ")
	fmt.Scanf("%d", &noPizza)

	var noSlices int
	fmt.Printf("How many slices per pizza? ")
	fmt.Scanf("%d", &noSlices)

	fmt.Printf("%d people with %d pizzas, each of %d slices\n", noPeople, noPizza, noSlices)
	fmt.Printf("Each person gets %d slices.\n", noSlices*noPizza/noPeople)
	fmt.Printf("There are %d leftover slices.\n", noSlices*noPizza%noPeople)

}
